include/windows/ifaddrs.h

```diff
@@ -24,8 +24,8 @@ struct ifaddrs {
 	struct sockaddr *ifa_addr;    /* Address of interface */
 	struct sockaddr *ifa_netmask; /* Netmask of interface */
 
-	struct sockaddr_in in_addr;
-	struct sockaddr_in in_netmask;
+	struct sockaddr in_addrs;
+	struct sockaddr in_netmasks;
```

Comment:
The size of struct sockaddr is the same as struct sockaddr_in.  You can't fit an IPv6 address into it.  The options here are:
- Use struct sockaddr_in6, so both ipv4 and ipv6 can fit
- Or use struct sockaddr_storage, which is large enough to hold both and defined specifically for this reason.
- Or create a union of sockaddr_in and sockaddr_in6.
The most maintainable option is to use sockaddr_storage.  That will consume slightly more memory than option 1.

Reply:
@shefty  Done, took sockaddr_storage.